dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -6,7 +6,6 @@
   {% set on_schema_change = incremental_validate_on_schema_change(config.get('on_schema_change'), default='ignore') %}
 
   {% set lf_tags_config = config.get('lf_tags_config') %}
-  {% set lf_inherited_tags = config.get('lf_inherited_tags') %}
```

Comment:
why this is removed? how an we pass lf_inherited_tags?

Reply:
I guess on the model level like other lf_tags_config
```
  +lf_tags_config:
    enabled: true
    tags:
      confidentiality: internal
    inherited_tags: []
```

Reply:
I realised after inspecting the code, but before `inherited_tags` where on the same level of lf_tags_config, so we should communicate that in the next release.